src/components/DirectAnswer.tsx

```diff
@@ -30,27 +30,19 @@ export interface DirectAnswerProps {
 export interface DirectAnswerCssClasses extends ThumbsFeedbackCssClasses {
   container?: string,
   container___loading?: string,
-  fieldValueTitle?: string,
-  featuredSnippetTitle?: string,
+  answer?: string,
```

Comment:
I'm still waiting on their response.  I messaged the slack channel a few days ago and bumped them this morning, but they have not gotten back to me.  

By answer, I was thinking more like the one word/sentence answer to the question which would be good to stand out to the user.  The description tells the background/context of the information, which is secondary to the quick answer.  

The two examples in Baigel's sandbox site are:

Featured snippet:  The short answer to the question "What is the best supermarket" is "Baigel's supermarkets are the best because they simply are". This is the main answer and will be in the `header` styled with `answer`. The context of the quote is given in the `body` and is styled by `description`.
![image](https://user-images.githubusercontent.com/59857107/175611550-8314190b-003d-491a-8b38-42409d5f39cd.png)

Field value:  The short answer to the question "What color are blueberries" is "Blue". This is the main answer and will be in the `body` styled with `answer`. The entity and value (i.e. the context) is given in the 'header` and is styled by `description`.
![image](https://user-images.githubusercontent.com/59857107/175612017-3d05c497-0be3-4016-9e35-2a89f50555d6.png)

This was the general motivation to try to go about it this way, and it helps preserve the layout of the direct answer component.

Reply:
If the name itself is the issue, then I can definitely rename it! haha